hibernate-spatial/src/test/java/org/hibernate/spatial/mapping/GeographyMappingTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.spatial.mapping;
+
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.metamodel.mapping.BasicValuedModelPart;
+import org.hibernate.persister.entity.EntityPersister;
+import org.hibernate.type.SqlTypes;
+import org.hibernate.type.descriptor.java.spi.JavaTypeRegistry;
+import org.hibernate.type.descriptor.jdbc.spi.JdbcTypeRegistry;
+
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.ServiceRegistry;
+import org.hibernate.testing.orm.junit.SessionFactory;
+import org.hibernate.testing.orm.junit.SessionFactoryScope;
+import org.junit.jupiter.api.Test;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import org.geolatte.geom.G2D;
+import org.geolatte.geom.Point;
+import org.geolatte.geom.codec.Wkt;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+@DomainModel(annotatedClasses = { GeographyMappingTest.PointEntity.class })
+@ServiceRegistry
+@SessionFactory
+public class GeographyMappingTest {
+
+	@Test
+	public void testSimpleEntity(SessionFactoryScope scope) {
+		final EntityPersister entityDescriptor = scope.getSessionFactory()
+				.getRuntimeMetamodels()
+				.getMappingMetamodel()
+				.getEntityDescriptor( PointEntity.class );
+		final JdbcTypeRegistry jdbcTypeRegistry = entityDescriptor.getFactory()
+				.getTypeConfiguration()
+				.getJdbcTypeRegistry();
+
+		final JavaTypeRegistry javaTypeRegistry = entityDescriptor.getFactory()
+				.getTypeConfiguration()
+				.getJavaTypeRegistry();
+
+		BasicValuedModelPart part = (BasicValuedModelPart) entityDescriptor.findSubPart( "location" );
+		assertThat( part.getJdbcMapping().getJdbcType(), equalTo( jdbcTypeRegistry.getDescriptor( SqlTypes.GEOGRAPHY ) ) );
+		scope.inTransaction(
+				s -> {
+					s.persist(
+							new PointEntity(
+									1,
+									"test",
+									(Point<G2D>) Wkt.fromWkt(
+											"SRID=4326;POINT(48.2083736 16.3724441)"
+									)
+							)
+					);
+					s.flush();
+					s.clear();
+
+					PointEntity pointEntity = s.find( PointEntity.class, 1 );
+					assertThat( pointEntity.location, is( notNullValue() ) );
```

Comment:
Is the `notNullValue()` assertion a little bit weak? Should we assert the 2D coordinates instead?